db/items.csv

```diff
@@ -37,5 +37,4 @@ Fleur de Lis Salt & Pepper Shakers,"Each Fleur De Lis Salt & Pepper Shaker is ha
 
 In French, fleur means flower, and lis means lily. The fleur-de-lis is a stylized lily or iris that is used as a decorative symbol often associated with the French monarchy. Louisville, named for France's King Louis XVI, is one of five cities and two states that have the fleur-de-lis in its flag or seal.",4500,0,http://cdn3.bigcommerce.com/s-4gmtl/images/stencil/100x100/products/335/582/fdld966_4__60242.1410795881.jpg?c=2
 Vintage Stone Cement Salt Jar,"Old style salt box, that can be either wall mounted or placed on a counter.  Typical of the ones seen in old kitchens in France and Northern Europe.  It's 3 inches in diameter and 3.5 inches high.",995,0,http://thecrossdesign.com/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/s/t/stone_cement_jar_2_1024x1024.jpg
-Stoneware & Cork salt & Pepper Shakers,This set of 2 pig shaped salt and pepper shakers will be happy to grace your dinner table.,999,0,https://www.touchofeurope.net/thumbnail.asp?file=assets/images/stonewarecorkpigsaltpeppershakerswhitesetof2.jpg&maxx=50&maxy=0
-Steampunk Pepper Mill,The hand-crank design of this finely handcrafted heirloom pepper mill was inspired by antique chef's tools.,9800,0,https://i.pinimg.com/564x/71/6f/67/716f67b26761c559586efd2d9acfd34f.jpg
\ No newline at end of file
+Steampunk Pepper Mill,The hand-crank design of this finely handcrafted heirloom pepper mill was inspired by antique chef's tools.,9800,0,https://i.pinimg.com/564x/71/6f/67/716f67b26761c559586efd2d9acfd34f.jpg

```

Comment:
I didn't really change anything here, I was just trying to find a higher resolution image for this one item, but it didn't work right.